splunk_connect_for_snmp/gopoller/auth.go

```diff
@@ -75,6 +77,10 @@ func getSecurityParameters(authData SnmpV3StringAuthData) (*gosnmp.UsmSecurityPa
 		PrivacyPassphrase:        authData.PrivacyPassphrase,
 		AuthoritativeEngineID:    authData.AuthoritativeEngineID,
 	}
+	useLogger := os.Getenv("USE_GO_LOGGER")
```

Comment:
maybe we could name it `ENABLE_GO_LOGGER` or something similar? 
Is it possible to set log level for this logger?
Could we add setting this flag to yaml? I think we could do section for go under
`poller.go` or `worker.poller.go`

Reply:
I can change the name to `ENABLE_GO_LOGGER` and add the configuration to values.yaml. When it comes to the log level, I found the following info in the gosnmp documentation: 
<img width="1149" alt="Screenshot 2024-02-01 at 13 51 26" src="https://github.com/splunk/splunk-connect-for-snmp/assets/108660584/0b7a390a-8fae-4f8d-bd2f-950e0790390d">

I also checked how this logger is used in code of gosnmp, and they don't distinguish any logging levels. They just simply log if the logger was configured

Reply:
Ok, so if it logs everything, we'll follow with that.